src/main/kotlin/com/github/sanity/kweb/KWeb.kt

```diff
@@ -34,19 +43,20 @@ class KWeb(val port: Int,
            val debug: Boolean = false,
            val refreshPageOnHotswap : Boolean = false,
            val plugins: List<KWebPlugin> = Collections.emptyList(),
-           val appServerConfigurator: (AppServer) -> Unit = {},
+           // val appServerConfigurator: (AppServer) -> Unit = {},
            val onError : ((List<StackTraceElement>, JavaScriptError) -> LogError) = { _, _ ->  true},
            val buildPage: RootReceiver.() -> Unit
 ) {
     companion object: KLogging()
 
-    private val server = AppServer(AppConfiguration(port = port))
+    private var server: NettyApplicationHost? = null
     private val clients: MutableMap<String, WSClientData>
     private val mutableAppliedPlugins: MutableSet<KWebPlugin> = HashSet()
     val appliedPlugins: Set<KWebPlugin> get() = mutableAppliedPlugins
 
     init {
-        appServerConfigurator.invoke(server)
+        // TODO How is this used?
+        // appServerConfigurator.invoke(server)
```

Comment:
...however, there should really be a more elegant way to do this, exposing the Wasabi AppServer in this way was a temporary fix.  Would appreciate your thoughts on a good way to allow the user to configure additional routes and their respective handlers.